ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/PhotoCompressionModifier.java

```diff
@@ -0,0 +1,39 @@
+package net.gini.android.vision.internal.camera.photo;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.support.annotation.NonNull;
+
+import java.io.ByteArrayOutputStream;
+
+/**
+ * @exclude
+ */
+class PhotoCompressionModifier implements PhotoModifier {
```

Comment:
Refactored PhotoEdit to be extensible less error prone, by abstracting away the modifications into classes.

This one does the compression.